pkg/rpc/server/server.go

```diff
@@ -465,18 +465,18 @@ func (s *Server) blockHashFromParam(param *request.Param) (util.Uint256, *respon
 		return hash, response.ErrInvalidParams
 	}
 
-	if _, err := param.GetStringStrict(); err == nil {
-		var err error
-		hash, err = param.GetUint256()
+	if _, err := param.GetIntStrict(); err == nil {
```

Comment:
...and avoid doing it here (just `s.blockHeightFromParam(param)`)?

Reply:
Fixed.